search-sentiments-map/routes/trends.js

```diff
@@ -238,16 +240,17 @@ function getGlobalTrends(trendsByCountry) {
     trendCountsArr.push({
       topic: topic,
       count: count,
-    })
+    });
   }
   // Sort trends in descending order of their counts.
   trendCountsArr.sort((trend, otherTrend) => {
     return otherTrend.count - trend.count;
-  })
+  });
 
   let globalTrends = [];
-  // Get the top 10 trends overall.
-  for (let i = 0; i < 10; i++) {
+  let numTopTrends = trendCountsArr.length < 10 ? trendCountsArr.length : 10;
```

Comment:
Create a constant for the max number(10).

Reply:
done